docs/800_TRANSMITS.md

```diff
@@ -0,0 +1,24 @@
+# Report Coordinator
+
+There are two methods to be supported for confirming transmits made it to chain.
+
+1. check that any key from a `Report` has a perform log received by the node
+2. (future) check that an OCR transmit log was received
```

Comment:
this has separate design considerations that I would recommend separating out from here